network_wrangler/models/roadway/tables.py

```diff
@@ -35,7 +35,7 @@ class RoadLinksTable(DataFrameModel):
     A: Series[int] = pa.Field(nullable=False, coerce=True)
     B: Series[int] = pa.Field(nullable=False, coerce=True)
     geometry: GeoSeries = pa.Field(nullable=False)
-    name: Series[str] = pa.Field(nullable=False)
```

Comment:
fixed in: https://github.com/wsp-sag/network_wrangler/commit/6828330b9228f78a882b3bd5c9cf7e2fc9b191ba